lib/sample_rate_converter/Kconfig

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config SAMPLE_RATE_CONVERTER
```

Comment:
Please make this a menuconfig to avoid these entries appearing in the top-level menu.

Reply:
Good point, done!